src/nle.c

```diff
@@ -26,8 +26,68 @@
 
 extern int unixmain(int, char **);
 
+short
+vt_font_attr_convert(TMTCHAR *c)
```

Comment:
While this is a nice idea, I'd opt to not go for this, as:

* NetHack itself already has an opinion for how to turn this thing you call "font" into a bitset.
* For ML purposes, bitsets are not too useful. Everything that can be enumerated should be -- and the colours for instance can.

Note that "bold" doesn't actually mean bold. Instead "bold" controls the "bright" bit of colors, e.g. doubles the value the foreground can have from 8 to 16.

So my suggestion would be to have a "tty_colors" observation that for "most frames" aligns with the "colors" observation (that contains only the dungeon, which may get occluded in the tty output).

For the "specials", NetHack uses these bitset values:

```
                        MG_CORPSE	0x01
			MG_INVIS	0x02
			MG_DETECT	0x04
			MG_PET		0x08
			MG_RIDDEN	0x10
			MG_STATUE	0x20
			MG_OBJPILE	0x40
```

We can't infer this from the terminal output here. However, NetHack also won't use most of what you include here: E.g., I don't think it will change the background color (except via `reverse`). It will not use blink or underline and likely won't use `invisible` either. As I said, `bold` is a "color" property. So is "dim" I think, which stands for "character is half-bright". So in effect I believe (may be slightly wrong) the only thing we get in addition is a single bit for "reverse", which we could include in the colors, e.g. as a sign bit in a int8?

Reply:
cool, i have changed font to color :) and we now mimic the info in `colors.h`